README.md

```diff
@@ -12,7 +12,7 @@ You can find us for developer discussion:
 - Linux with a modern C++17 friendly compiler
 - 64 bit
 - CMake v3.2+
-- OpenGL 4.5+ or Vulkan 1.1.92.1+
+- OpenGL 4.5+ or Vulkan 1.1.121.2+
```

Comment:
There is no need to change the minimum version requirement in the README as decaf only uses features from vulkan making 1.1.92.1 the minimum version.

Reply:
ok